ebay-frontend/src/app/app-routing.module.ts

```diff
@@ -6,7 +6,7 @@ import { UpcomingEventsScreenComponent } from './upcoming-events-screen/upcoming
 import { PastEventsScreenComponent } from './past-events-screen/past-events-screen.component';
 import { LandingPageComponent } from './landing-page/landing-page.component';
 import { HomeComponent } from './home/home.component';
-
+import { CreateEventsScreenComponent } from './create-events-screen/create-events-screen.component';
```

Comment:
`CreateEventsScreen` should just be called `CreateEventScreen` because you're only creating one event at a time.